web-components/src/components/inputs/TextField.tsx

```diff
@@ -71,7 +65,7 @@ const useStyles = makeStyles<Theme, StyleProps>((theme: Theme) => ({
       color: theme.palette.primary.light,
       position: props.errors ? 'absolute' : 'inherit',
       lineHeight: Object.keys(props.form.errors).length > 0 ? 1.3 : 1.66,
-      bottom: 0,
+      bottom: props.errors ? theme.spacing(-5) : 0,
```

Comment:
I believe these sorts of `spacing` values were set to match the height of the MUI `helperText` which shows error messages so there wasn’t a layout thrash when error messages come in.  this could be fine, but it’s probably also okay to just remove all of thsoe properties (if it’s easier, which it might not be) and have the form shift a little when there are errors. That seems fairly common.

Alternatively, we could always render `helperText` and throw a `{‘ ‘}` in its content when there isn’t actual text - the error messages target that element I believe.

No need to change this here, just something i’ve thought about so wanted to comment. Long-term, it might actually be best to just have a big “reafactor styles” story where we rip everything custom out and apply styles top-down, starting at the theme level and prioritizing `sx` for one-offs. Seems like a lot, but I actually think once the initial theme styles were set and normalzied, it might not be a ton of work because we’d be working with the framework, whereas right now we seem to fight against it a good deal 😄